jax/_src/lax/parallel.py

```diff
@@ -180,6 +180,16 @@ def pmin(x, axis_name, *, axis_index_groups=None):
                          axis_index_groups=axis_index_groups)
   return tree_util.tree_unflatten(treedef, out_flat)
 
+def pargmax(x, axis_name):
+  if isinstance(axis_name, (tuple, list)):
+    raise TypeError(f"pargmax only accepts a single axis, got {axis_name}")
+  return pargmax_p.bind(x, axis=axis_name)
+
+def pargmin(x, axis_name):
+  if isinstance(axis_name, (tuple, list)):
+    raise TypeError(f"pargmin only accepts a single axis, got {axis_name}")
```

Comment:
While we can definitely get away with primitives not supporting multiple axes in the user-facing API, we definitely cannot do that internally. `xmap` usually replaces each user-facing axis with multiple internal axes.

Reply:
Well, we could if we change the `xmap` lowering strategy not to use the split-axis approach :P

But point taken, I forgot about this constraint.